frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -756,6 +756,7 @@ export const surveyLogic = kea<surveyLogicType>([
                 urlMatchType: values.urlMatchTypeValidationError,
             }),
             submit: (surveyPayload) => {
+                actions.editingSurvey(false)
```

Comment:
I noticed that we weren't turning off edit mode on survey submission, which I think we should since the edits are complete when the survey has been submitted.